bedrock/firefox/templates/firefox/new/scene1_linux.html

```diff
@@ -0,0 +1,49 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/new/quantum" %}
+
+{% extends "firefox/new/scene1.html" %}
+
+{% block page_title_prefix %} {{_('Download Mozilla Firefox')}} {% endblock %}
+{%- block page_title -%} {{_('for Linux')}} {%- endblock -%}
```

Comment:
I probably should have caught this on the Windows page as well, but I think it would be better to make this one string and omit the prefix. Breaking the string apart like this can make l10n more difficult since some languages might need to reorder the words (still no idea if there's any intent to localize these pages, but we can act as if there is). 

So just `{% block page_title %}{{ _('Download Mozilla Firefox for Linux') }}{% endblock %}` and the same for Mac and Windows.